pipelines/rh-advisories/README.md

```diff
@@ -22,6 +22,14 @@ the rh-push-to-registry-redhat-io pipeline.
 | verify_ec_task_bundle           | The location of the bundle containing the verify-enterprise-contract task                                                          | No       | -                                                         |
 | taskGitUrl                      | The url to the git repo where the release-service-catalog tasks to be used are stored                                              | Yes      | https://github.com/konflux-ci/release-service-catalog.git |
 | taskGitRevision                 | The revision in the taskGitUrl repo to be used                                                                                     | No       | -                                                         |
+| updateComponentSBOM             | Update and upload component-level SBOMs                                                                                            | Yes      | false                                                     |
```

Comment:
This should be defined in the data file, not a pipeline parameter. Pipeline parameters are provided by the release-service operator and so anything that is supposed to be provided by the user is defined in RP or RPA's `data` field.

Reply:
ok, cool